src/sagemaker/serve/model_format/mlflow/utils.py

```diff
@@ -392,17 +397,62 @@ def _select_container_for_mlflow_model(
     )
 
 
-def _validate_input_for_mlflow(model_server: ModelServer) -> None:
+def _validate_input_for_mlflow(model_server: ModelServer, deployment_flavor: str) -> None:
     """Validates arguments provided with mlflow models.
 
     Args:
         - model_server (ModelServer): Model server used for orchestrating mlflow model.
+        - deployment_flavor (str): The flavor mlflow model will be deployed with.
 
     Raises:
     - ValueError: If model server is not torchserve.
     """
-    if model_server != ModelServer.TORCHSERVE:
+    if model_server != ModelServer.TORCHSERVE and model_server != ModelServer.TENSORFLOW_SERVING:
```

Comment:
Should this be `and` operator?

Reply:
Yes, we only support two model servers for mlflow models right now.

Reply:
Got it.

Reply:
I am not following, the idea here is that if model server is not in [TORCHSERVE, TENSORFLOW_SERVING], then raise exception. If it's or, then if you provide either TORCHSERVE or TENSORFLOW_SERVING, it will meet the requirement of the other half of the branch and result in exception.

Reply:
I think I was thinking in terms of

not (A or B) = (not A) and (not B)

Reply:
I see, I don't like parentheses, so I will go with not A and not B

Reply:
think left is supposed to be faster but anyway